apps/emqx_auth_jwt/test/emqx_auth_jwt_SUITE.erl

```diff
@@ -164,3 +168,129 @@ t_check_claims_kid_in_header(_) ->
     Result0 = emqx_access_control:authenticate(Plain#{password => Jwt}),
     ct:pal("Auth result: ~p~n", [Result0]),
     ?assertMatch({ok, #{auth_result := success, jwt_claims := _}}, Result0).
+
+t_check_jwt_acl(_) ->
+    application:set_env(emqx_auth_jwt, verify_claims, [{sub, <<"value">>}]),
+    application:stop(emqx_auth_jwt), application:start(emqx_auth_jwt),
```

Comment:
Maybe it's safer to do this setup (and later clean-up) in `{init,end}_per_testcase`.

Reply:
✅